nmtwizard/framework.py

```diff
@@ -960,7 +997,16 @@ def preprocess_into_model(
         if parent_model:
             config["parent_model"] = parent_model
         config["model"] = model_id
-        config["modelType"] = "preprocess"
+        if model_type == "standalone":
+            config["modelType"] = "standalone"
+            data = config.pop("data", None)
+            if data:
+                if parent_model_type == "checkpoint":
+                    config["standalone_data"] = data
+                else:
+                    config["used_data"] = data
```

Comment:
If I understand correctly, this field is just for information. It that's the case, I suggest moving it inside the "build" section.